src/utilities/typography.scss

```diff
@@ -18,194 +18,285 @@
-  font-size: $h3-size-mobile !important;
-  font-weight: $font-weight-light !important;
+  font-size: var(--h3-size-mobile, $h3-size-mobile) !important;
+  font-weight: var(--base-text-weight-light, $font-weight-light) !important;
 
-  @include breakpoint(md) { font-size: $h3-size !important; }
+  @include breakpoint(md) {
+    font-size: var(--h3-size, $h3-size) !important;
+  }
 }
 
 // Smallest text size
 /* Set the font size to ${#h6-size} */
-.text-small { font-size: $h6-size !important; } // 12px
+.text-small {
+  font-size: var(--h6-size, $h6-size) !important;
+}
+
+// 12px
 
 /* Large leading paragraphs */
 .lead {
   // stylelint-disable-next-line primer/spacing
   margin-bottom: 30px;
-  font-size: $h3-size;
-  font-weight: $font-weight-light;
+  font-size: var(--h3-size, $h3-size);
+  font-weight: var(--base-text-weight-light, $font-weight-light);
 }
 
 // Line-height variations
 // Close to commonly used line-heights. Most line-heights
 // combined with type size equate to whole pixels.
 // Will be improved with future typography scale updates.
 // Responsive line-height
-@each $breakpoint, $variant in $responsive-variants {
+@each $breakpoint,
+$variant in $responsive-variants {
   @include breakpoint($breakpoint) {
+
     /* Set the line height to ultra condensed */
-    .lh#{$variant}-condensed-ultra { line-height: $lh-condensed-ultra !important; }
+    .lh#{$variant}-condensed-ultra {
+      line-height: $lh-condensed-ultra !important;
+    }
+
     /* Set the line height to condensed */
-    .lh#{$variant}-condensed { line-height: $lh-condensed !important; }
+    .lh#{$variant}-condensed {
+      line-height: $lh-condensed !important;
+    }
+
     /* Set the line height to default */
-    .lh#{$variant}-default { line-height: $lh-default !important; }
+    .lh#{$variant}-default {
+      line-height: $lh-default !important;
+    }
+
     /* Set the line height to zero */
-    .lh#{$variant}-0 { line-height: 0 !important; }
+    .lh#{$variant}-0 {
+      line-height: 0 !important;
+    }
   }
 }
 
 // Text alignments
 // Responsive text alignment
-@each $breakpoint, $variant in $responsive-variants {
+@each $breakpoint,
+$variant in $responsive-variants {
   @include breakpoint($breakpoint) {
+
     /* Text align to the right */
-    .text#{$variant}-right { text-align: right !important; }
+    .text#{$variant}-right {
+      text-align: right !important;
+    }
+
     /* Text align to the left */
-    .text#{$variant}-left { text-align: left !important; }
+    .text#{$variant}-left {
+      text-align: left !important;
+    }
+
     /* Text align to the center */
-    .text#{$variant}-center { text-align: center !important; }
+    .text#{$variant}-center {
+      text-align: center !important;
+    }
   }
 }
 
 // Text styles
 /* Set the font weight to normal */
-.text-normal { font-weight: $font-weight-normal !important; }
+.text-normal {
+  font-weight: $font-weight-normal !important;
```

Comment:
I don't mind if it's a formatting fix, but wanted to double check you didn't mean to also add CSS vars for these lines?

Reply:
I actually added CSS vars to these SCSS vars instead. It seemed like a safe change to make since no math would occur here 😄 

```
// Font weights
$font-weight-bold: var(--base-text-weight-medium, 600) !default;
$font-weight-semibold: var(--base-text-weight-semibold, 500) !default;
$font-weight-normal: var(--base-text-weight-normal, 400) !default;
$font-weight-light: var(--base-text-weight-light, 300) !default;
```